tests/display/test_alert.py

```diff
@@ -0,0 +1,52 @@
+from typing import Callable
+
+import pytest
+from dash import Dash, Input, Output, dcc
+
+import dash_mdc_neptune as mdc
+
+
+@pytest.fixture
+def dash_app() -> Callable[[str], Dash]:
+    def app_factory(text_value: str) -> Dash:
+        app = Dash(name=__name__)
+        app.layout = mdc.Dashboard(
+            children=[
+                mdc.Page(
+                    children=mdc.Section(
+                        children=mdc.Box(
+                            children=[
+                                dcc.Input(id="input_text", value=text_value),
+                            ],
+                        ),
+                        cards=[{"title": "Card"}],
+                    ),
+                ),
+                mdc.Alert(id="alert"),
+            ]
+        )
+
+        @app.callback(
+            Output(component_id="alert", component_property="message"),
+            Input(component_id="input_text", component_property="value"),
+        )
+        def update_alert(value: str) -> str:
+            return value
+
+        return app
+
+    return app_factory
+
+
+@pytest.mark.parametrize(
+    "text_value,alert_value",
+    [
+        ("Test alert", "Test alert"),
+    ],
+)
+def test_render_navbar(dash_duo, dash_app, text_value, alert_value):
+    dash_duo.start_server(dash_app(text_value))
+
+    dash_duo.wait_for_text_to_equal("#alert", alert_value)
```

Comment:
💡 

Probably more defensive test if input text is set first, then check the correct alert display in the HTML document.